src/core/ui.js

```diff
@@ -24,6 +24,14 @@ styleElement.classList.add("removeOnSave");
 
 document.head.appendChild(styleElement);
 
+let body = $('body');
+let fileIssueLink = $('<a href="https://github.com/w3c/respec/issues/new?title=&body=" id="fileIssueLink">file an issue related to the text selected</a>');
```

Comment:
Nit: add css class "removeOnSave" to the element.

Reply:
Sure